lib/Dashboard/CalendarWidget.php

```diff
@@ -30,6 +30,7 @@
 use DateTimeImmutable;
 use OCA\Calendar\AppInfo\Application;
 use OCA\Calendar\Service\JSDataService;
+use OCA\DAV\CalDAV\CalendarImpl;
```

Comment:
this is a private API

Reply:
yeah, still means this will introduce an inter-app dependency. we should avoid that at all cost.

Reply:
`IGetTimezone` is just fine, right?